eman2/protocols/protocol_tomo_boxing.py

```diff
@@ -82,6 +88,45 @@ def _readSetOfCoordinates3D(self, jsonBoxDict, tomo, coord3DSetDict):
 
                 self._readCoordinates3D(box, tomo, coord3DSet)
 
+    def _coordinates2json(self, inputCoor):
+        cwd = os.getcwd()
+        infoDir = pwutils.join(cwd, 'info')
+        self._leaveWorkingDir()
+        fnInputCoor = 'extra-%s_info.json' % pwutils.removeBaseExt(self.inputTomo.getFileName())
+        pathInputCoor = pwutils.join(infoDir, fnInputCoor)
+        if not exists(pathInputCoor):
+            pwutils.makePath(infoDir)
+        f = open(pathInputCoor, 'w')
+        initFile = '{\n"boxes_3d": [\n\n],\n"class_list": {\n"0": {\n"boxsize": 32,\n"name": "particles_00"\n}\n}\n}'
+        f.write(initFile)
+        f.close()
+        firstItem = inputCoor.getFirstItem()
+        linCoor = '[%d, %d, %d, "manual", 0.0, 0]' % (firstItem.getX(), firstItem.getY(), firstItem.getZ())
+        r = open(pathInputCoor, "r")
+        contents = r.readlines()
+        r.close()
+        contents.insert(2, linCoor)
+        w = open(pathInputCoor, "w")
+        contents = "".join(contents)
+        w.write(contents)
+        w.close()
+        idx = 1
+        for coor in inputCoor.iterCoordinates():
+            if idx == 1:
```

Comment:
If you actually need to write the first coordinate outside the loop, I would something a bit smarter here. You can find some ideas [here](https://stackoverflow.com/questions/10079216/skip-first-entry-in-for-loop-in-python)

Reply:
Solved in https://github.com/scipion-em/scipion-em-eman2/pull/35/commits/1a2b4d12be94ff10faa5b5eac8373d8028d61f89